apps/dg/resources/scripts.rhtml

```diff
@@ -1,14 +1,13 @@
 <% content_for :page_javascript do %>
 <% if config.enable_google_analytics %>
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-ZYRHG9RN3K"></script>
```

Comment:
I want to note for posterity that I'm aware hard-coding the Google Analytics ID value here is a little at odds with our use of `DG.GOOGLE_ANALYTICS_ID` below. Unfortunately, there doesn't seem to be a straightforward way to use a dynamic value for this first `script` tag. It will at least do no harm to CODAP functionality or to the Concord Consortium's tracking of CODAP usage via GA.